app/models/state_file_nc_intake.rb

```diff
@@ -106,7 +106,7 @@ class StateFileNcIntake < StateFileBaseIntake
   enum eligibility_ed_loan_cancelled: { no: 0, yes: 1 }, _prefix: :eligibility_ed_loan_cancelled
   enum eligibility_ed_loan_emp_payment: { no: 0, yes: 1 }, _prefix: :eligibility_ed_loan_emp_payment
   enum paid_extension_payments: { unfilled: 0, yes: 1, no: 2 }, _prefix: :paid_extension_payments
-  enum out_of_country: { no: 0, yes: 1 }, _prefix: :out_of_country
+  enum out_of_country: { unfilled: 0, yes: 1, no: 2 }, _prefix: :out_of_country
```

Comment:
eek, i don't think we can just shift the enum like that if there are records on production with this column filled out. if we do want to do it this way i think we'd have to backfill the values so that anyone currently with a 0 has a 2 filled out instead. if you don't want to do the backfill you can do this: 
`enum out_of_country: { no: 0, yes: 1, unfilled: 2 }, _prefix: :out_of_country`
and then in a migration:
`change_column_default :state_file_nc_intakes, :out_of_country, from: 0, to: 2`

Reply:
discussed with jenny that this column is not being used in prod (it is behind the `extension_period` flipper flag) but if it was, definitely requires a migration!